pkg/providers/aws/provider_blobstorage.go

```diff
@@ -156,6 +156,49 @@ func (p *BlobStorageProvider) CreateStorage(ctx context.Context, bs *v1alpha1.Bl
 		},
 	}
 
+	// Adding tags to s3
+	p.Logger.Infof("bucket %s found, Adding tags to bucket", *bucketCreateCfg.Bucket)
+
+	// get the environment from the CR
+	defaultOrganizationTag := bs.Spec.Env.Value
+
+	// new session but don't need credentials
+	s3svctag := s3.New(session.Must(session.NewSession(&aws.Config{Region: aws.String(stratCfg.Region)})))
```

Comment:
We have a helper func for creating a session `createS3Session`

Reply:
@ciaranRoche tried it and it gives a 403, as the comment says don't need credentials, but should probably say doesn't work with credentials.

Reply:
It sounds like a permissions issue, AWS can be poor at times with the error codes. Have you tired adding any extra permissions to our credentials https://docs.aws.amazon.com/AmazonS3/latest/dev/using-with-s3-actions.html

Its just so strange it allows for tagging with out credentials at all.

Reply:
yea it was a strange to me as well until I found this example https://github.com/aws/aws-sdk-go/blob/master/service/s3/examples_test.go#L1704L1720, the only thing our implementation complained about was it was missing a region with the old new.Session function , so change to the non deprecated method for session and added a region.  Haven't changed anything to the permissions.